tests/cpp/merge_mps/merge_mps_test.cpp

```diff
@@ -0,0 +1,213 @@
+#include <fmt/format.h>
+#include <gtest/gtest.h>
+
+#include "LoggerStub.h"
+#include "RandomDirGenerator.h"
+#include "InMemoryWriter.h"
+#include "antares-xpansion/benders/merge_mps/MergeMPS.h"
+
+size_t StandardLp::appendCNT = 0;
+
+using namespace std::string_literals;
+
+class MergeMPSTest : public ::testing::Test {
+protected:
+    void SetUp() override {
+        tmp_dir_ = CreateRandomSubDir(std::filesystem::temp_directory_path());
+        logger_ = std::make_shared<Xpansion::Test::LoggerNOOPStub>();
+        writer_ = std::make_shared<Xpansion::Test::InMemoryWriter>();
+
+        options_.SOLVER_NAME = "COIN";
+        options_.STRUCTURE_FILE = (tmp_dir_ / "structure_file.txt").string();
+        options_.OUTPUTROOT = tmp_dir_.string();
+    }
+
+    void TearDown() override {
+    }
+
+    void createMasterProblem() {
+        std::ofstream master_problem(tmp_dir_ / "master.mps"s);
+        master_problem << R"(NAME          MASTER  FREE
```

Comment:
Could you add a comment somewhere to write the mathematical problem that it represents ?